### PR TITLE
Fixes #18346: Make the group option to include the Rudder server or not in the group more clear

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18346

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
                 <span class="ion ion-checkmark-round check-icon"></span>
               </label>
               <label for="typeQuery" class="form-control">
-                Include Rudder server components
+                Include Rudder root server
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html:21]: It is not only root, but all node with a server component: so nodes having rudder-reports, rudder-db, rudder-webapp or rudder-server-root
But not the relays

Telling root is not correct - maybe something in the line
"Include Rudder server components (web interface, database)"

**Review by ncharles — CHANGES_REQUESTED** [review]
It is not only root server

File: webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html:21]: nobody knows what "server components" are (I didn't think it's documented anywhere). I really think it should say "root". 

```
Include Rudder root server components (nodes with Rudder application or database)
```

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html:21]: "application" is maybe too broad, though

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html:21]:
```
Include Rudder root server components (nodes with Rudder web application or database)
```

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html:21]: image

**Comment by fanf** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**